features/env.feature

```diff
@@ -0,0 +1,4 @@
+Feature: The Tuist environment
+    Scenario: Installing tuist from source
+        Given that tuist is available
+        Then tuistenv should succeed in installing "master"
```

Comment:
This step might introduce some flakiness to the test suite. We can leave it but we should keep an eye on it.

Reply:
I have the same concern, I wasn't sure of how else we could test it (we need to install _something_)